app/app.go

```diff
@@ -111,10 +111,6 @@ func (a *App) InitServer() {
 		}
 		a.srv.RunJobs()
 	})
-	a.accountMigration = a.srv.AccountMigration
-	a.ldap = a.srv.Ldap
-	a.notification = a.srv.Notification
-	a.saml = a.srv.Saml
```

Comment:
@jespino - this looks to be an artifact of the fakeapp migration. If you see L68-71, they are already being set there. Setting them again to the same value does not seem to be intentional. Let me know.

Reply:
Ok it seems like removing these lines broke the EE tests. Looking into it.

Reply:
Some fields were missing in `ServerConnector`. Hopefully it's fixed now.